openshift_tools/monitoring/zagg_sender.py

```diff
@@ -27,6 +27,14 @@
 import os
 import yaml
 
+# Add Hawkular sender backend (Optional)
```

Comment:
> p.s. If we move to using hawkular why we need to send data to zabbix ?

Because we can't replace zabbix without some time of overlapping (e.g. for feature parity, testing, etc.)